content/posts/this-month-in-hax/2024-07.md

```diff
@@ -0,0 +1,70 @@
+---
+author: "Lucas Franceschino"
+title: "This month in hax: July 2024"
+date: "2024-08-12"
+tags: ["this-month-in-hax"]
+ShowToc: false
+ShowBreadCrumbs: false
+---
+
+That's the second blog post of our "This Month in hax" series, and
+this month, in july, we merged 32 PRs! 🎉
```

Comment:
Let's assume that this is going to be a monthly thing now and we don't want to count how often we did it already 😄 

```suggestion
In July, we merged 32 PRs! 🎉
```